apps/web/app/[lang]/blog/article/[slug]/components/Interesting.tsx

```diff
@@ -0,0 +1,29 @@
+import Image from 'next/image'
+import Link from 'next/link';
+import { InterestingProps } from '../Article.types';
+
+
+export const Interesting: React.FC<InterestingProps> = ({blogPost}) => {
+    return <>
+            { blogPost && <div className="pb-6 border-b border-gray-medium flex-col justify-start items-start gap-8 inline-flex">
```

Comment:
Please check for blogPost first and return null if there is no value.
```suggestion
            if (!blogPost) return null;
            <div className="pb-6 border-b border-gray-medium flex-col justify-start items-start gap-8 inline-flex">
```